src/custom-resource-loader.js

```diff
@@ -31,6 +31,11 @@ export class CustomResourceLoader extends ResourceLoader {
         this._active = true;
         this._requestStats = requestStats;
         this._logging = logging;
+
+        // Remove any unused files from the fetch_package cache. We do this
+        // before we start any requests to make sure we don't overfill the
+        // cache with all the new data we download.
+        flushUnusedCache();
```

Comment:
I think this is in the wrong place.

We do request files from the `CustomResourceLoader` when inside JSDOM, but before that, we also request files at the very start of the request that we use to set up the JSDOM environment. I think this flushing therefore needs to be either moved so that it occurs for both cases, or moved to the request handling part instead of here (probably the latter).

Reply:
Ahh! Yes, doing this in `server.js` makes a lot more sense, moving it there now...